libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pivotTable/PivotTableDescriptor.ts

```diff
@@ -123,7 +124,9 @@ export class PivotTableDescriptor extends BaseChartDescriptor implements IVisual
 function factoryNotationForAttributeColumnWidthItem(obj: IAttributeColumnWidthItem): string {
     const { attributeIdentifier, width } = obj.attributeColumnWidthItem;
     const { value: widthValue, allowGrowToFit } = width;
-    return allowGrowToFit
-        ? `newWidthForAttributeColumn(${attributeIdentifier}, ${widthValue}, true)`
-        : `newWidthForAttributeColumn(${attributeIdentifier}, ${widthValue})`;
+    // must be isNil to keep 0 values in
```

Comment:
Nitpick" this is a bit confusing comment (I read it as "the value must be Nil") :)